
 Makefile                 |   26 +++++++++++++++-----------
 scripts/kconfig/Makefile |    8 --------
 2 files changed, 15 insertions(+), 19 deletions(-)

diff -Nurp linux-3.8-rc2/Makefile linux-3.8-rc2.mrp/Makefile
--- linux-3.8-rc2/Makefile	2013-01-06 19:31:22.988241619 +0200
+++ linux-3.8-rc2.mrp/Makefile	2013-01-06 19:33:10.320119303 +0200
@@ -513,8 +513,9 @@ ifeq ($(KBUILD_EXTMOD),)
 # Carefully list dependencies so we do not try to build scripts twice
 # in parallel
 PHONY += scripts
-scripts: scripts_basic include/config/auto.conf include/config/tristate.conf
-	$(Q)$(MAKE) $(build)=$(@)
+#scripts: scripts_basic include/config/auto.conf include/config/tristate.conf
+#	$(Q)$(MAKE) $(build)=$(@)
+scripts:
 
 # Objects we will link into vmlinux / subdirs we need to visit
 init-y		:= init/
@@ -831,7 +832,8 @@ prepare0: archprepare FORCE
 	$(Q)$(MAKE) $(build)=.
 
 # All the preparing..
-prepare: prepare0
+#prepare: prepare0
+prepare:
 
 # Generate some files
 # ---------------------------------------------------------------------------
@@ -1013,13 +1015,15 @@ endif # CONFIG_MODULES
 CLEAN_DIRS  += $(MODVERDIR)
 
 # Directories & files removed with 'make mrproper'
-MRPROPER_DIRS  += include/config usr/include include/generated          \
-                  arch/*/include/generated
-MRPROPER_FILES += .config .config.old .version .old_version $(version_h) \
-		  Module.symvers tags TAGS cscope* GPATH GTAGS GRTAGS GSYMS \
-		  signing_key.priv signing_key.x509 x509.genkey		\
-		  extra_certificates signing_key.x509.keyid		\
-		  signing_key.x509.signer
+MRPROPER_DIRS  +=
+MRPROPER_FILES +=
+#MRPROPER_DIRS  += include/config usr/include include/generated          \
+#                  arch/*/include/generated
+#MRPROPER_FILES += .config .config.old .version .old_version $(version_h) \
+#		  Module.symvers tags TAGS cscope* GPATH GTAGS GRTAGS GSYMS \
+#		  signing_key.priv signing_key.x509 x509.genkey		\
+#		  extra_certificates signing_key.x509.keyid		\
+#		  signing_key.x509.signer
 
 # clean - Delete most, but leave enough to build external modules
 #
@@ -1040,7 +1044,7 @@ clean: archclean vmlinuxclean
 #
 mrproper: rm-dirs  := $(wildcard $(MRPROPER_DIRS))
 mrproper: rm-files := $(wildcard $(MRPROPER_FILES))
-mrproper-dirs      := $(addprefix _mrproper_,Documentation/DocBook scripts)
+mrproper-dirs      := $(addprefix _mrproper_,Documentation/DocBook)
 
 PHONY += $(mrproper-dirs) mrproper archmrproper
 $(mrproper-dirs):
diff -Nurp linux-3.8-rc2/scripts/kconfig/Makefile linux-3.8-rc2.mrp/scripts/kconfig/Makefile
--- linux-3.8-rc2/scripts/kconfig/Makefile	2012-12-11 05:30:57.000000000 +0200
+++ linux-3.8-rc2.mrp/scripts/kconfig/Makefile	2013-01-06 19:33:10.320119303 +0200
@@ -12,26 +12,18 @@ Kconfig := Kconfig
 endif
 
 xconfig: $(obj)/qconf
-	$< $(Kconfig)
 
 gconfig: $(obj)/gconf
-	$< $(Kconfig)
 
 menuconfig: $(obj)/mconf
-	$< $(Kconfig)
 
 config: $(obj)/conf
-	$< --oldaskconfig $(Kconfig)
 
 nconfig: $(obj)/nconf
-	$< $(Kconfig)
 
 oldconfig: $(obj)/conf
-	$< --$@ $(Kconfig)
 
 silentoldconfig: $(obj)/conf
-	$(Q)mkdir -p include/generated
-	$< --$@ $(Kconfig)
 
 localyesconfig localmodconfig: $(obj)/streamline_config.pl $(obj)/conf
 	$(Q)mkdir -p include/generated
